client/src/components/pages/application/Tasks/index.js

```diff
@@ -55,97 +47,123 @@ const Tasks = () => {
     });
     if (UserId) {
       axios.get(`/api/v1/applicants/${UserId}`).then(({ data: { user } }) => {
-        if (technicalTasks === '') setTechnicalTasks(user.technicalTasks);
+        if (user.technicalTasksLinks) {
+          setTechnicalTasksLinks(user.technicalTasksLinks);
+        }
+        setTechnicalTasks(user.technicalTasks);
```

Comment:
there's problem in technicalTasks value it didn't change to `true` in database when you finish all tasks . I suggest to use Boolean state for it .